packages/components/molecules/f-rating/src/assets/scss/rating.scss

```diff
@@ -0,0 +1,37 @@
+// Add styles here so it can be injected first via vue.config.js.
+@use '@justeat/fozzie/src/scss/fozzie' as f;
+
+.c-rating {
+    display: flex;
+}
+
+.c-rating-stars {
+    margin: 0;
+    padding: 0;
+    position: relative;
+    display: inline-block;
+}
+.c-rating-stars-icons,
+.c-rating-stars-icon {
+    display: flex;
+}
+
+.c-rating-star {
```

Comment:
should this be `c-rating-star-small` instead to make it consistent with the other classes?

Reply:
Was part of a discussion with @ashleynolan. Had initially had that but we decided to use the default class as the smaller variant.

Reply:
Sounds good, might as well make 'small' the default value for `starRatingSize` then 😄